src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -120,38 +131,9 @@ public String getEIBackendSubscriptionAddress() {
         LOG.info("Got HTTP Request with method GET.\nUrlSuffix: " + eiBackendAddressSuffix
                 + "\nForwarding Request to EI Backend with url: " + newRequestUrl);
 
-        HttpClient client = HttpClients.createDefault();
         HttpGet eiRequest = new HttpGet(newRequestUrl);
 
-        String jsonContent = "";
-        HttpResponse eiResponse = null;
-        try {
-            eiResponse = client.execute(eiRequest);
-
-            InputStream inStream = eiResponse.getEntity().getContent();
-            BufferedReader bufReader = new BufferedReader(new InputStreamReader(inStream, "UTF-8"));
-            for (String line = bufReader.readLine(); line != null; line = bufReader.readLine()) {
-                jsonContent += line;
-            }
-            bufReader.close();
-            inStream.close();
-        } catch (IOException e) {
-            LOG.error("Forward Request Errors: " + e);
-        }
-
-        LOG.info("EI Http Reponse Status Code: " + eiResponse.getStatusLine().getStatusCode()
-                + "\nEI Recevied jsonContent:\n" + jsonContent + "\nForwarding response back to EI Frontend WebUI.");
-
```

Comment:
Why removing the HTTP Return code from debug Message? Good to know when debugging the web-server.